spell_check/spell_ignore_md.txt

```diff
@@ -0,0 +1,73 @@
+Ctags
+annot
+ci
+cmd
+codecov
+commandline
+cov
+functiondefextractor
+gh
+html
+https
+img
+io
+jscpd
+md
+mutmut
+opensource
+philips
+pydoc
+pylint
+pytest
+svg
+travis
+txt
+xlsx
+Brijesh
+Krishnan
+Reddy
+Sannihith
+brijesh
+krishnank
+philips
+reddyp
+sannihith
+Koninklijke
+NONINFRINGEMENT
+Philips
+https
+philips
+sublicense
+www
+codecov
+cov
+ctags
+excellmedia
+gnuwin
+htm
+http
+https
+io
+mutmut
+py
+pylint
+pypa
+pytest
+sourceforge
+txt
+unittest
+wiki
+wikipedia
+CHANGELOG
+repo
+src
+MERCHANTABILITY
+SuppressWarnings
+excelfile
+Supress
+TypeScript
+dataframe
+github
+thirdparty
+xlrd
+xlsxwriter
```

Comment:
Would you like to consider some refactoring on MD file as well. Some of the spell check ignore could be better read well when its modified in md file. (thirdparty can be third party)

Reply:
Corrected some of these.